README.md

```diff
@@ -44,8 +44,8 @@ A curated collection of the best stuff from the Tauri ecosystem and community.
 
 ### Guides
 
-- [Introduction](https://tauri.app/about/intro/) ![officially maintained] - Official introduction to Tauri.
-- [Getting Started](https://tauri.app/v1/guides/getting-started/prerequisites/) ![officially maintained] - Official getting started with Tauri docs.
+- [Introduction](https://v2.tauri.app/start/) ![officially maintained] - Official introduction to Tauri.
```

Comment:
v2 is not stable yet, we should wait on this change

Reply:
i ghosted this pr long enough, please take another look now that it's stable 😂 (i can't merge my own prs in this repo :( )